tests/test_download.py

```diff
@@ -89,15 +103,41 @@ def create_spacetime_layer(self):
 
         return TiledRasterLayer.from_numpy_rdd(LayerType.SPACETIME, rdd, metadata)
 
+    def create_spatial_layer(self):
```

Comment:
could this be moved to datacube_fixtures.py, or even reuse an existing function from somewhere?

Reply:
Done. If Ok, I can merge